src/__tests__/Navigation.test.js

```diff
@@ -0,0 +1,10 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navigation from '../components/Navigation';
+
+it('renders correctly', () => {
+  const navigation = renderer
```

Comment:
- [ ] Great job on the design and functionlaity so far but I think you missed very important requirement so I would higly recommend to add it so you should filter the data that you retrieve from the api so you can use a search bar to search for specific crypto coin for example or any possible solution for filtering the data you can use so please add it as it is one of the important techincal requirements 🙏🏻✨💯🚀🚀

![image](https://user-images.githubusercontent.com/58553711/210929472-5dd86853-6105-464f-8809-33639bf16ae7.png)